fixtures/permissions/allow-person-playbook.yaml

```diff
@@ -0,0 +1,14 @@
+---
+# yaml-language-server: $schema=../../config/schemas/permission.schema.json
+apiVersion: mission-control.flanksource.com/v1
+kind: Permission
+metadata:
+  name: allow-user-playbook-run
+spec:
+  description: allow user john to run any playbook
+  subject:
+    person: john@doe.com
+  action: playbook:run
```

Comment:
@moshloop should we make this a list?

---
requires changing column `permissions.action` from `text`->`jsonb`